spec/support/database_cleaner.rb

```diff
@@ -0,0 +1,26 @@
+require 'database_cleaner'
```

Comment:
Have you been able to update Hound? You'll need a rubocop file to remove this comment. Otherwise, you should be fixing it